NewForce-at-Mountwest/bangazon-api-farfalle
# Scaffolded react app

# Description

Scaffolded react application. Created a components folder containing applicationviews.js, bangazon.js, bangazon.css, and a nav folder/component. Also created a modules folder containing a file named APIManager.

Added navbar.

## Related Ticket(s)
13

## Steps to Test
Fetch the branch sb-react, cd into bangazon-react, cd into the node-modules folder, and type npm install, npm install. When that is finished, type npm install react-router-dom. You should also use the command npm install react-bootstrap bootstrap. Open the app in visual code and 
check that all named files are there.


# Checklist:
